test/fuzz/README.md

```diff
@@ -0,0 +1,30 @@
+# Fuzzing
+
+ Fuzzing or fuzz testing is an automated software testing technique that involves providing invalid, unexpected, or random data as inputs to a computer program. A fuzzing target is defined for running the tests. These targets are defined in the test/fuzz directory and run using the instructions given below:
```

Comment:
Change the line length to 80 (why isn't markdownlint catching this...?).

Reply:
Could it be that we don't check files in nested folders? :scream:

Reply:
running markdown lint on this file isn't giving any errors for line length. I tried adding new line breaks to change length but then it is giving trailing space issues here. When I tried adding br tags which is another way it was also giving linting error. What should be the way to resolve this since I don't receive any error for this rule.

Reply:
MDL had obsolete config file, which ignored line length. It'll be fixed here: https://github.com/metal3-io/ip-address-manager/pull/252

Reply:
OK, it has merged. Please rebase and also squash commits.